### ISSUE TITLE
[Arcs] Outrage timing of Empathy for All in Breaking World action

### ISSUE THREAD

**Issue by turmoilbyrd** [LederCards/cards#48]
[Breaking World](https://cards.ledergames.com/card/ARCS-F1103B) is an action that _harms_ all buildings. What happens if it was thwarted (after the battles, the Planet Breaker no longer controls the planet). Do they still trigger outrage due to [Empathy for All](https://cards.ledergames.com/card/ARCS-F1409)?

**Comment by turmoilbyrd** [comment]
Reading the errata for Breaking World, maybe the outrage for Empathy for all happens at the time buildings are destroyed? (Now Step 2 post errata)

**Comment by jdyearsley** [comment]
Yes, even initiating the action counts as harming. (This also prevents Imperial-on-Imperial violence in the presence of Imperial ships, etc.) I'll add an FAQ.
